resteasy-core/src/main/java/org/jboss/resteasy/plugins/interceptors/GZIPEncodingInterceptor.java

```diff
@@ -37,20 +44,15 @@ public void finish() throws IOException
       }
    }
 
-   public static class CommittedGZIPOutputStream extends CommitHeaderOutputStream
+   public static class CommittedGZIPAsyncOutputStream extends CommitHeaderAsyncOutputStream
    {
-      protected CommittedGZIPOutputStream(final OutputStream delegate, final CommitCallback headers)
+      protected CommittedGZIPAsyncOutputStream(final AsyncOutputStream delegate)
       {
-         super(delegate, headers);
+         super(delegate, null);

```

Comment:
I decided to pass `null` to the `super` constructor instead of receiving a `null` `headers` argument in the constructor.
But now both constructors receive a `headers` argument again to keep backwards compatibility.